code/game/objects/items/weapons/tools.dm

```diff
@@ -358,7 +358,7 @@
 	var/safety = user:eyecheck()
 	if(ishuman(user))
 		var/mob/living/carbon/human/H = user
-		var/obj/item/organ/eyes/E = H.internal_organs_by_name["eyes"]
+		var/obj/item/organ/internal/eyes/E = H.get_int_organ(/obj/item/organ/internal/eyes)

```

Comment:
I was thinking of making an argument against this, but actually...
This would make robotic internal organ repair surgery for IPCs (table, screwdriver, crowbar, screwdriver, crowbar) more relevant than an occasional rarity that comes up if you find one that's been esworded to death, or something. So sure, this sounds fine, especially since simple external welding doesn't do the eye check. I think it's worth a try, at least.
Though, it seems slightly inconsistent given that station cyborgs don't need to worry about welding protection.

If people hate it, I can always look into getting things back to how they are.